sa/database.go

```diff
@@ -98,6 +81,10 @@ func recombineURLForDB(dbConnect string) (string, error) {
 
 	dsnVals.Set("parseTime", "true")
 
+	// Required to make UPDATE return the number of rows matched,
+	// instead of the number of rows changed by the UPDATE.
+	dsnVals.Set("clientFoundRows", "true")
```

Comment:
Yep, that's the case. Being a MySQL consultant is pretty lucrative because it's very popular and has plenty of weird, legacy defaults.